### ISSUE TITLE
Using Generative Adverserial Loss Instead of L2 loss

### ISSUE THREAD

**Issue by azurespace** [kaonashi-tyc/Rewrite#1]
Your work is very impressive to me. I think this work may be applied for font style copying among different languages. 

But as you said in the front page, I've found some of images that your network generated are blurry. That is one of characteristic of L2(mse) loss. 

So I think it would be worth to try GAN. I guess you already know but I just write a brief explanation about it. When you train a GAN, you use and train another network(called discriminator) simultaneously that predicts image whether it is original or generated one. 

This is a tensorflow implementation of GAN for Superresolution implemented in Tensorflow.
https://github.com/buriburisuri/SRGAN

**Comment by EricCartmanofHIT** [comment]
![step_6000](https://cloud.githubusercontent.com/assets/17721372/24757816/e498fe0e-1b12-11e7-84c0-a836c1174a5e.png)
![step_60002](https://cloud.githubusercontent.com/assets/17721372/24757825/ec724edc-1b12-11e7-934f-9a7740599767.png)

I tried GAN and VGG for loss function,but i found it hard to balance the result between the two result shown above.

**Comment by kaonashi-tyc** [comment]
@azurespace consider fixed https://github.com/kaonashi-tyc/zi2zi

Can we close this issue right now? :)
